end_to_end/signal_handler_test.go

```diff
@@ -40,13 +55,18 @@ var _ = Describe("Signal handler tests", func() {
 				rng := rand.New(rand.NewSource(time.Now().UnixNano()))
 				time.Sleep(time.Duration(rng.Intn(1000)+500) * time.Millisecond)
 				_ = cmd.Process.Signal(unix.SIGINT)
+
+				for outScanner.Scan() {
+					stdout.WriteString(outScanner.Text())
+					stdout.WriteByte('\n')
+				}
 			}()
-			output, _ := cmd.CombinedOutput()
-			stdout := string(output)
-			Expect(stdout).To(ContainSubstring("Received an interrupt signal, aborting backup process"))
-			Expect(stdout).To(ContainSubstring("Cleanup complete"))
-			Expect(stdout).To(Not(ContainSubstring("CRITICAL")))
-			timestamp := getBackupTimestamp(stdout)
+			_ = cmd.Run()
+			output := stdout.String() + stderr.String()
+			Expect(output).To(ContainSubstring("Received an interrupt signal, aborting backup process"))
+			Expect(output).To(ContainSubstring("Cleanup complete"))
+			Expect(output).To(Not(ContainSubstring("CRITICAL")))
+			timestamp := getBackupTimestamp(output)
```

Comment:
how about
```suggestion
			output, _ := cmd.CombinedOutputWithStdoutAndStderr()
			stdout := string(output)
			Expect(stdout).To(ContainSubstring("Received an interrupt signal, aborting backup process"))
			Expect(stdout).To(ContainSubstring("Cleanup complete"))
			Expect(stdout).To(Not(ContainSubstring("CRITICAL")))
			timestamp := getBackupTimestamp(stdout)
```
where
```go
func (c *Cmd) CombinedOutputWithStdoutAndStderr() ([]byte, error) {
	if c.Stdout == nil {
		return nil, errors.New("exec: Stdout is not set")
	}
	if c.Stderr == nil {
		return nil, errors.New("exec: Stderr is not set")
	}
	err := c.Run()
	return c.Stdout.Bytes() + c.Stderr.Bytes(), err
}
```

Reply:
The Writer does not have a Bytes() field.

Reply:
rewrite by strings.Builder, main idea is reduce diff